audius-cli

```diff
@@ -744,7 +744,6 @@ def upgrade(ctx, branch):
 
             if service == "discovery-provider":
                 ctx.invoke(launch_chain)
```

Comment:
how does chain not have this same problem?

Reply:
chain doesn't get pruned because it's in the discovery docker compose file